iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -189,4 +192,38 @@ public KeyValuePairTree KEY_VALUE_PAIR_WITH_EQUALS() {
       f.keyValuePairEquals(b.token(STRING_LITERAL), b.token(Punctuator.EQU), b.token(STRING_LITERAL))
     );
   }
+
+  public CmdTree CMD() {
+    return b.<CmdTree>nonterminal(DockerLexicalGrammar.CMD).is(
+      f.cmd(
+        b.token(DockerKeyword.CMD),
+        b.optional(EXEC_FORM())
```

Comment:
I'm supprised that the actual command is optional, but it seems valid.

Reply:
Yes, docker for `CMD` produce:
`CMD ["/bin/sh" "-c" ""]`